dynawo/3rdParty/adept/adept-chain-2.0.5.sh

```diff
@@ -88,25 +87,25 @@ install_adept() {
     tar -xzf $ADEPT_ARCHIVE -C $BUILD_DIR
   fi
   cd $BUILD_DIR/$ADEPT_DIRECTORY
-  if [ "$COMPILER" = "GCC" ]; then
-    CXXFLAGS_ADEPT=-Wall
-  elif [ "$COMPILER" = "CLANG" ]; then
-    CXXFLAGS_ADEPT=-Weverything
-  else
-    error_exit "COMPILER environment variable needs to be GCC or CLANG."
-  fi
-  export CXXFLAGS_ADEPT="$CXXFLAGS_ADEPT $CXX_STDFLAG -fPIC"
   if [ "$DYNAWO_LIBRARY_TYPE" = "SHARED" ]; then
     ADEPT_LIBRARY_TYPE_OPTION="--disable-static --enable-shared"
   else
     ADEPT_LIBRARY_TYPE_OPTION="--enable-static --disable-shared"
   fi
-  if [ "${BUILD_TYPE}" = "Release" ]; then
-    ./configure "CXXFLAGS=$CXXFLAGS_ADEPT -O3" --prefix=$INSTALL_DIR CC=$C_COMPILER CXX=$CXX_COMPILER --disable-openmp $ADEPT_LIBRARY_TYPE_OPTION
+  if [ "$(echo "$DYNAWO_CXX11_ENABLED" | tr '[:upper:]' '[:lower:]')" = "yes" -o "$(echo "$DYNAWO_CXX11_ENABLED" | tr '[:upper:]' '[:lower:]')" = "true" -o "$(echo "$DYNAWO_CXX11_ENABLED" | tr '[:upper:]' '[:lower:]')" = "on" ]; then
+    if [ "$BUILD_TYPE" = "Debug" ]; then
+      ./configure "CXXFLAGS=-g -O0 -fPIC" --prefix=$INSTALL_DIR CC=$DYNAWO_C_COMPILER CXX=$DYNAWO_CXX_COMPILER --disable-openmp $ADEPT_LIBRARY_TYPE_OPTION
+    else
+      ./configure "CXXFLAGS=-O3 -fPIC" --prefix=$INSTALL_DIR CC=$DYNAWO_C_COMPILER CXX=$DYNAWO_CXX_COMPILER --disable-openmp $ADEPT_LIBRARY_TYPE_OPTION
+    fi
   else
-    ./configure "CXXFLAGS=$CXXFLAGS_ADEPT -g -O0" --prefix=$INSTALL_DIR CC=$C_COMPILER CXX=$CXX_COMPILER --disable-openmp $ADEPT_LIBRARY_TYPE_OPTION
+    if [ "$BUILD_TYPE" = "Debug" ]; then
+      ./configure "CXXFLAGS=-g -O0 -fPIC -std=c++98" --prefix=$INSTALL_DIR CC=$DYNAWO_C_COMPILER CXX=$DYNAWO_CXX_COMPILER --disable-openmp $ADEPT_LIBRARY_TYPE_OPTION
```

Comment:
Can't we put this -std=... in an environment variable to avoid this complex if?

Reply:
I changed it